kite-data/kite-data-core/src/main/java/org/kitesdk/data/View.java

```diff
@@ -109,6 +111,15 @@
   public Class<E> getType();
 
   /**
+   * Get the schema of entities contained in this {@code View}.
+   *
+   * @return the schema of entities contained in this view
+   *
+   * @since 1.1.0
+   */
+  public Schema getSchema();
```

Comment:
Another option that potentially solves both this and the dataset cast issue below (need two copies of a dataset with different types) is to make the instance class type available only on views. Then the dataset simply has a schema that is the same as the descriptor's schema. You can then get a view with a different type or a view with a different schema, but the underlying dataset is always `Dataset<GenericRecord>`.

The `Datasets` class would be simpler, replacing all methods that take a type with something like this:

```java
View<Specific> data = Datasets.load(uri).asType(Specific.class);
```